pkg/ipamd/datastore/data_store.go

```diff
@@ -349,59 +372,36 @@ func (ds *DataStore) ReadBackingStore() error {
 	ds.lock.Lock()
 	defer ds.lock.Unlock()
 
-	eniIPs := make(ENIPool)
-	eniPrefixes := make(ENIPool)
-
+	eniCidrs := make(ENIPool)
```

Comment:
seems this eniCidrs is never used. If it's indeed like that, can we change AvailableIPv4Cidrs from map to slice.

Reply:
Yeah :)